ajax/libs/redom/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "RE:DOM",
+  "description": "Tiny UI library",
+  "homepage": "https://redom.js.org",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/pakastin/redom"
+  },
+  "author": "Juha Lindstedt",
+  "license": "ISC",
```

Comment:
@pakastin I think the license is `"MIT"`. Ref: https://github.com/pakastin/redom/blob/master/LICENSE